src/ground-truth.rkt

```diff
@@ -26,7 +26,7 @@
 ;; The first element of that function's output tells you if the input is good
 ;; The other elements of that function's output tell you the output values
 (define (make-search-func pre specs ctxs)
-  (define fns (compile-specs (cons pre specs) (context-vars (car ctxs))))
+  (define fns (compile-specs (cons pre specs) (context-vars (car ctxs)) (context-repr (car ctxs))))
```

Comment:
I think this actually also maybe isn't quite right, since it assumes all outputs use the same `repr`. This won't be correct if an expression uses mixed binary32/binary64 code. That said, I'm not too worried about this because I think it'll need to be fixed a different way anyway.